qaframework-bdd-tests/src/test/resources/features/refapp-2.x/16-openConceptLab/openConceptLab.feature

```diff
@@ -0,0 +1,34 @@
+Feature: OCL Subscription module
+
+Background:
+    Given a user clicks on configure Metadata link from home page
+    When the system loads the configure metadata page
+     And a user clicks Manage OCL link from configure metadata page
+    Then the system loads Open Concept Lab page
+
+  @selenium
+  @openConceptLab
+  Scenario: OCL MANAGEMENT
+     #user story: setup subscription
+     When a user clicks on Setup subscription button
+     Then the system loads Subscription page
+      And a user enters the URL of a new released dictionary
+      And a user enters the Token url
+      And a user clicks on the Save Changes button
+     Then the system loads Open Concept Lab Success page
+      And a user clicks import from Subscription server button
+      Then the API should be displayed on the previous imports
+     #user story:Edit subscription
+     When a user clicks edit subscription button
+    Then the system loads Subscription page
+      And a user edits the URL of a released dictionary
+      And a user edits the Token url
+      And a user clicks on the Save Changes button
+      And the system loads Open Concept Lab page
+      And a user clicks import from Subscription server button
+     Then the API should be displayed on the previous imports
+     #user story: Unsubscribe
```

Comment:
```js
 #user story: Unsubscribe
     When a user clicks edit subscription button
     Then the system loads Subscription page
      And a user clicks unsubscribe button
     Then the system loads Subscription page 
```
Does it require one to click in edit subscription button in order to unsubscribe?

Reply:
@kdaud yes.

Reply:
What is the user story accomplishing?

Reply:
@kdaud Kindly you can try it out on the reference application to get a clear picture but without it the system can't run normally much less when the information is in the stored in the database.

Reply:
What is the user story all about?

Reply:
@kdaud Kindly I don't seem to understand what you are asking neither where you are heading. Do you mind being clear and to the point on what you are trying to put across?

Reply:
The user story 
```js
#user story: Unsubscribe
```
what does it accomplish? In other words what can a user do with the feature?